ykrt/src/trace/hwt/mapper.rs

```diff
@@ -60,8 +60,8 @@ impl<'a> HWTMapper {
     ///     ...
     ///
     /// During codegen LLVM may remove the unconditional jump and simply place bb1 and bb2
-    /// consecutively, allowing bb1 to fall-thru to bb2. In the eyes of the PT block decoder, a
-    /// fall-thru does not terminate a block, so whereas LLVM sees two blocks, PT sees only one.
+    /// consecutively, allowing bb1 to fall-thru to bb2. In the eyes of hwtracer, a full-thru does
```

Comment:
Typo here, but not yours. `full-thru` -> `fall-thru`

Reply:
Fixed in ef5225b.